packages/flutter/lib/src/material/paginated_data_table.dart

```diff
@@ -476,7 +482,7 @@ class PaginatedDataTableState extends State<PaginatedDataTable> {
         .toList();
       footerWidgets.addAll(<Widget>[
         Container(width: 14.0), // to match trailing padding in case we overflow and end up scrolling
-        Text(localizations.rowsPerPageTitle),
+        Text(widget.rowsPerPageTitle ?? localizations.rowsPerPageTitle),
```

Comment:
I think it might be best to approach customizing this through localizations. Looking at the rest of the footer items using localizations, if we expose this one, why not all of them? Enumerating so many properties on a widget when they can be accessed and overridden via an object like localizations is not typically a change we would make here.